### PR TITLE
Add documentation to run unit-test through Intellij

### PR DESCRIPTION
**Pull request by mikepapadim**
#### Description

Add entry to documentation to explain the cofiguration for running unit-tests through Intellij.

#### Backend/s tested

Mark the backends affected by this PR.

- [ ] OpenCL
- [ ] PTX
- [ ] SPIRV

#### OS tested

Mark the OS where this PR is tested.

- [ ] Linux
- [ ] OSx
- [ ] Windows

#### Did you check on FPGAs?

If it is applicable, check your changes on FPGAs.

- [ ] Yes
- [ ] No

----------------------------------------------------------------------------

### CODE REVIEW

File: docs/source/ide-integration.rst [hunk 1]
```diff
@@ -47,7 +47,7 @@ Run and Debug TornadoVM with Intellij
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Normal maven lifecycle goals like *package* and *install* will not
-result a succefull build for TornadoVM.
+result a successful build for TornadoVM.
```
Comment (stratika) [inline docs/source/ide-integration.rst:50]:
```suggestion
result in a successful build for TornadoVM.
```

File: docs/source/ide-integration.rst [hunk 2]
```diff
@@ -58,8 +58,8 @@ Build/Run Configuration
 
 -  **In the Project Tab:**
 
-   -  The Project SDK uses the same java version as the project (e.g. Java 17).
-   -  The Project language level is using the same java version (e.g. Java 17 with Lambdas, type annotations etc.).
+   -  The Project SDK uses the same java version as the project (e.g. Java 21).
+   -  The Project Language Level is using the same java version (e.g. Java 21 (Preview).
```
Comment (stratika) [inline docs/source/ide-integration.rst:62]: Let's update `line 67` with Java 21, for consistency

File: docs/source/ide-integration.rst [hunk 3]
```diff
@@ -115,8 +115,8 @@ Output should be something similar to this:
 
 .. code:: bash 
 
-   /PATH_TO_JDK/jdk1.8.0_131/bin/java
```
Comment (stratika) [inline docs/source/ide-integration.rst:118]: `Lines 69-71` the `right click > create tornadovm [package]` does not to exist any more. Can we please update also those instructions?

File: docs/source/ide-integration.rst [hunk 4]
```diff
@@ -115,8 +115,8 @@ Output should be something similar to this:
...
-   -server -XX:-UseJVMCIClassLoader -XX:-UseCompressedOops -Djava.ext.dirs=/home/michalis/Tornado/tornado/bin/sdk/share/java/tornado -Djava.library.path=/home/michalis/Tornado/tornado/bin/sdk/lib -Dtornado.load.api.implementation=uk.ac.manchester.tornado.runtime.tasks.TornadoTaskGraph -Dtornado.load.runtime.implementation=uk.ac.manchester.tornado.runtime.TornadoCoreRuntime -Dtornado.load.tornado.implementation=uk.ac.manchester.tornado.runtime.common.Tornado -Dtornado.load.device.implementation.opencl=uk.ac.manchester.tornado.drivers.opencl.runtime.OCLDeviceFactory -Dtornado.load.device.implementation.ptx=uk.ac.manchester.tornado.drivers.ptx.runtime.PTXDeviceFactory
```
Comment (stratika) [inline docs/source/ide-integration.rst:119]: `Lines 76-90` please update the `cmake` version and the `jdk-8` profile to match the current version/profiles.

**Review by stratika — CHANGES_REQUESTED** [review]

**Review by jjfumero — CHANGES_REQUESTED** [review]

File: docs/source/ide-integration.rst [hunk 5]
```diff
@@ -131,14 +131,55 @@ Then, add your own parameters similar to the following:
...
+   $ tornado --printJavaFlags
+
+Output should be something similar to this:
+
+.. code:: bash
+
+   /PATH_TO_JDK/java/current/bin/java
+    -server -XX:-UseCompressedOops -XX:+UnlockExperimentalVMOptions -XX:+EnableJVMCI -XX:-UseCompressedClassPointers --enable-preview -Djava.library.path=...
+
+You need to copy from ``-server`` to end.
```
[the remark this answers is not in the document]
Comment (andrii0lomakin) [reply docs/source/ide-integration.rst:164]: If I had gotten it correctly, this option ` -XX:-UseCompressedOops` would not have been needed after migration to Java FF API.

[the remark this answers is not in the document]
Comment (jjfumero) [reply docs/source/ide-integration.rst:164]: Exactly, there is no need for this flag any more.

**Comment by jjfumero** [comment]
@mikepapadim , is this PR still going?

**Comment by mikepapadim** [comment]
> @mikepapadim , is this PR still going?

let me do another pass, so we can merge it

**Comment by stratika** [comment]
I will close this PR, after merging PRs #582 #583

**Pull request closed without merging**